Jenkinsfile.internal

```diff
@@ -0,0 +1,82 @@
+properties([buildDiscarder(logRotator(numToKeepStr: '20'))])
+
+pipeline {
+    agent {
+        label 'ubuntu-1804'
+    }
+
+    options {
+        skipDefaultCheckout(true)
+    }
+
+    environment {
+        TAG = tag()
+        BUILD_TAG = tag()
+    }
+
+    stages {
+        stage('Build') {
+            parallel {
+                stage("Validate") {
+                    agent {
+                        label 'ubuntu-1804'
+                    }
+                    steps {
+                        dir('src/github.com/docker/cnab-to-oci') {
+                            checkout scm
+                            ansiColor('xterm') {
+                                sh 'make -f docker.Makefile lint'
+                            }
+                        }
+                    }
+                    post {
+                        always {
+                            deleteDir()
```

Comment:
Do we need to clean the images here too since the linter creates an image?

Reply:
You're right, fxed